quic/s2n-quic-core/src/recovery/bandwidth/estimator/tests.rs

```diff
@@ -0,0 +1,328 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::*;
+use crate::{
+    frame::ack_elicitation::AckElicitation,
+    inet::ExplicitCongestionNotification,
+    path,
+    time::{Clock, NoopClock},
+};
+
+#[test]
+fn on_packet_sent_timestamp_initialization() {
```

Comment:
nit: as these are testing very implementation/rfc specific details can you add a description of what you are testing what you expect to happen.

+1 if you want to annotate with Setup, Trigger, Expectation

Reply:
Added some more explanation for this